qase-javascript-commons/changelog.md

```diff
@@ -1,3 +1,10 @@
+# qase-javascript-commons@2.0.0-beta.11
+
+## What's new
+
+Fixed an issue when you use the `useV2` option in the reporter's configuration. This options doesn't work correctly and always uses the v1 API.
+Fixed an issue when the step has an attachment. The attachment was not uploaded to the Qase.
```

Comment:
* The `useV2` option in the reporter's configuration will now enable using the experimental v2 API.
  Before this fix, v1 API was used despite the configuration.

* Attachments from test steps will now be uploaded to Qase.
  Before this fix, the reporter uploaded only the attachments made outside of any step scope.